client/components/jetpack-idc-notice/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Notice } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { addQueryArgs } from '@wordpress/url';
+
+const isInJetpackIdc = window.hasOwnProperty(
+	'JP_IDENTITY_CRISIS__INITIAL_STATE'
+);
+
+const JetpackIdcNotice = () => {
+	return (
+		isInJetpackIdc && (
+			<Notice
+				status="error"
+				isDismissible={ false }
+				className="wcpay-jetpack-idc-notice"
+			>
+				{ __(
+					'Your site is currently in Safe Mode.',
+					'woocommerce-payments'
+				) }
+				<span>&nbsp;</span>
+				<a href={ addQueryArgs( '/wp-admin/index.php' ) }>
```

Comment:
If this suggestion is accepted https://github.com/Automattic/jetpack/pull/22266, I will use WCPay settings page, i.e. `wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments`.